src/services/billing/cost-management/widgets/lib/widget-data-helper.ts

```diff
@@ -218,3 +224,23 @@ export const getTooltipText = (categoryKey, valueKey, money, disablePercentage =
     }
     return `{${categoryKey}}: [bold]${money}[/] ({${valueKey}.percent.formatNumber('#.00')}%)`;
 };
+
+
+const getWidgetDefaultOptions = async (widgetId?: string) => {
```

Comment:
```suggestion
const getWidgetDefaultOptions = async (widgetId?: string): Promise<WidgetOptions|undefined> => {
```
return 타입에 대한 명세가 있어야 함수를 사용하는 쪽에서 잘못된 값이 오는 경우에 대비할 수 있을 것 같아요!